### DISCUSSION TITLE
Order Management System - Sprint 3

### DISCUSSION THREAD

**Discussion by raghusaripalli** [raghusaripalli/GatorGiveaway#88] [General]
As Buyer and Seller modules are in place, let's build the Order Management System for [Sprint 3](https://github.com/raghusaripalli/GatorGiveaway/projects/3)

In the front end, we would want to add Cart, a feature providing the users to add, remove and update the count of items to the cart.
Also, an Order page, to confirm the order, List all previous and upcoming orders and cancel the order

In the backend, we would support those functionalities by exposing APIs like listing all orders, place and cancelling an order.

This thread is open to ideas, design discussions, and work sharing.

**Comment by tejasri555** [comment]
Amazon uses something like a message queue system for Order Management System and places all the orders in the queue.  The buyer looks into the queue and reviews them. Once the buyer confirms, the order will be confirmed.  Can we try this approach?

**Comment by ksmSujay95** [comment]
As part of Frontend, will add a cart icon at the top nav bar and show up the count of items added to the cart and redirect it to the order page.
